cli_reference/basic_cli_operations.adoc

```diff
@@ -206,6 +206,25 @@ results is filtered by that value.
 $ oc get <object_type> [<object_name>]
 ----
 
+The `oc get <object_type>` provides the `-o`, `--output` options to modify the output format.
+----
+$ oc get <object_type> -o
+$ oc get <object_type> --output
```

Comment:
To use one line and to cover overall format options, it is better to write it as:
oc get <object_type> [<object_name>] -o|--output json|yaml|wide|custom-columns=...|custom-columns-file=...|go-template=...|go-template-file=...|jsonpath=...|jsonpath-file=...